chart/README.md

```diff
@@ -541,14 +541,63 @@ Some example commands are shown below.
    sonatype/nexus-iq-server-ha --version <version>
    ```
 
+### Autoscaling
+
+IQ HA helm chart includes support for Kubernetes Horizontal Pod Autoscaling (HPA). With this enabled you can set the
+cluster to automatically scale up/down based on the resource demand such as CPU and Memory utilization.
+
+Note: When setting auto-scaling parameters please make sure to have sufficient hardware resources available in the
+underlying nodes meet the max pod demands.
+HPA is disabled by default. If you want to enable it, you need to set the `hpa.enabled` parameter to `true`.
+
+   ```
+   --set hpa.enabled=true
+   --set hpa.resources.memory.enabled=true
```

Comment:
--set hpa.resources.memory.enabled=true not needed

Reply:
commit: a5f2f4e88459cc6b5845096a1615dd6d2ead03a9